spec/features/dashboard_spec.rb

```diff
@@ -3,6 +3,23 @@
 require 'rails_helper'
 
 describe 'Punches Dashboard', type: :feature do
+  describe 'navigation bar' do
+    let(:user) { create(:user) }
+
+    context 'when the user is logged in' do
+      before do
+        sign_in user
+      end
+
+      it 'has the dropdown option on the navigation bar' do
+        visit root_path
+
+        # find('.resume').click
+        expect(page).to have_link(I18n.t('activerecord.models.education_experience.one'), href: education_experiences_path)
```

Comment:
use the text directly in the test, this way if the translation has a typo or is wrong, is easier to notice